samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -48,12 +48,23 @@ Configuration
 Angle of arrival mode
 =====================
 
-To build this sample with AoA mode only, set :makevar:`EXTRA_CONF_FILE` to the :file:`overlay-aoa.conf` file using the respective :ref:`CMake option <cmake_options>`.
+This sample builds the angle of arrival (AoA) mode by default.
 
 For more information about configuration files in the |NCS|, see :ref:`app_build_system`.
```

Comment:
I don't think this line is needed since no configuration files are now mentioned in this section.

Reply:
But the default mode used in the sample is not documented, it was not obvious for me (not being the original implementer of the sample).

1. sample did not mention the supported modes
2. I did not want to remove section, in case there were references (may be there are none?)

Reply:
I was more referring to Line 55 specifically, since Angle of Arrival section now doesn't mention setting CMake options.

Reply:
As the AoA is default no special CMake options need to be set, only when needing AoD the said CMake options in that section is applicable.

Reply:
```suggestion
```
So this line can go.